src/sdl2/video.rs

```diff
@@ -355,14 +355,14 @@ pub mod gl_attr {
 
 #[derive(Copy, Clone, Eq, PartialEq, Hash, Debug)]
 pub struct DisplayMode {
-    pub format: u32,
+    pub format: pixels::PixelFormatEnum,
```

Comment:
Could you `use pixels::PixelFormatEnum` and only use `PixelFormatEnum` here and everywhere else where you made a change ?